src/routes.rs

```diff
@@ -111,3 +117,42 @@ where
         }
     }
 }
+
+async fn log_request_response_body(
+    request: Request,
+    next: Next,
+) -> Result<impl IntoResponse, Response> {
+    let request = buffer_request_body(request).await?;
+    let response = next.run(request).await;
+    let response = buffer_response_body(response).await?;
+
+    Ok(response)
+}
+
+async fn buffer_request_body(request: Request) -> Result<Request, Response> {
```

Comment:
this should only happen if sth specific is set e.g. logger option to trace

in our logger settings we can write `live_compositor::routes=debug` to enable debugging in that one file, check if is there a way to provide a custom values that can enable specific log.

If above is not possible then lets enable that only when trace loggin is on